extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js

```diff
@@ -1,10 +1,11 @@
 /*global define*/
 define('ext.wikia.adEngine.provider.gpt.adSizeFilter', [
 	'ext.wikia.adEngine.adContext',
+	'wikia.abTest',
 	'wikia.document',
 	'wikia.log',
 	require.optional('wikia.breakpointsLayout')
-], function (adContext, doc, log, breakpointsLayout) {
+], function (adContext, abTest, doc, log, breakpointsLayout) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 5 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)